module/cli_controller.py

```diff
@@ -100,14 +107,14 @@ def make_path_string(path_list):
     """
     Prints the info and any children or parameters of the node.
     """
-    @staticmethod
-    def print_help(node):
+    def print_help(self):
```

Comment:
what is the reason this did not stay static?

Reply:
This uses self now. This was changed so now the function does not require any arguments and that makes it a lot simpler in the global_commands dict.